### ISSUE TITLE
conf files should be under /etc, log files should be in /var/log

### ISSUE THREAD

**Issue by yurivict** [chipmuenk/pyFDA#80]
Currently, it installs conf files into the python package directory:
```
	/usr/local/lib/python2.7/site-packages/pyfda/pyfda_log.conf
	/usr/local/lib/python2.7/site-packages/pyfda/pyfda_log_debug.conf
```

Then the app fails:
```
IOError: [Errno 13] Permission denied: '/usr/local/lib/python2.7/site-packages/pyfda/pyfda.log'
```
because this is a wrong location for the log file.

**Comment by chipmuenk** [comment]
Thanx for reporting - I hadn't noticed because I only use pyfda in develop mode where the directories are writeable.

Do you have a suggestion how to select a logging directory across operating systems? I could hardcode directories depending on the OS but I'm not sure what's best under Windows (C:\Temp? \C:\Users\Default\AppData\Roaming?) and I have no clue about OS X.

**Comment by yurivict** [comment]
I am on FreeBSD. On FreeBSD and Linux, this patch works fine:

```
s|package_data={'pyfda': \['pyfda_log.conf', 'pyfda_log_debug.conf',|package_data={'pyfda': [|
s|data_files = \[|data_files = [('${PREFIX}/etc', ['pyfda/pyfda_log.conf', 'pyfda/pyfda_log_debug.conf'])|"
```

I never use Windows or MacOS, so I have no idea where things are installed there. MacOS possibly also has /etc.

**Comment by chipmuenk** [comment]
The latest PR & merge to master should fix the issue - please test and give me feedback!

**Comment by chipmuenk** [comment]
fixed by #96.
